packages/sanes-browser-extension/src/index.tsx

```diff
@@ -34,20 +34,18 @@ const render = (
   );
 };
 
-type PersonaContexType = [GetPersonaResponse, boolean];
+// REVIEW getHasStoredPersona() has a Promise<boolean> signature but
+// hasStoredPersona gets type boolean | null
```

Comment:
Odd, I got pure boolean type.

Reply:
For variable length arrays, `Promise.all` expects the same type for each component.

Reply:
Really? Hmm, I got proper typing, maybe this is VC bug I am not sure.

Reply:
Did you restart VS Code after installing the latest TS version? Otherwise you still run the old TS 3.5 type server with the VS Code.

Reply:
VSCode won't highlight this issue unless you install the [nightly extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-typescript-next).

Reply:
You can choose if you want to use the TS version shipped with VS Code or the one installed locally in your project. See

<img width="1478" alt="Bildschirmfoto 2019-12-09 um 14 27 37" src="https://user-images.githubusercontent.com/2603011/70439431-4e549400-1a90-11ea-87d5-3e88d57c5128.png">

Reply:
Yes, you are right. Your solution with `as const` solves this issue.